<launch>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/camera/color/image_raw"/>
    <param name="autosize" value="true" />
  </node>
  <node name="soft_gripper_interface"
        pkg="lab4_pkg"
        type="soft_gripper_interface.py"
        output="screen">
  </node>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch" />
</launch>
